### PR TITLE
Add pouring sounds to pouring reagents

### PR DESCRIPTION
**Pull request by TheBoondock**
## About The Pull Request
Adds liquid sloshing sound when pouring reagents between containers.
Not sure if this is loud enough tho gimme feedback
## Why It's Good For The Game
More sound cues are pretty good when youre pouring reagents

https://github.com/tgstation/tgstation/assets/92416224/174a07cf-439b-47a3-b1fe-6eda3fcec32e
## Changelog
:cl:
sound: added liquid pouring sounds to pouring of any reagents
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/holder/holder.dm [hunk 1]
```diff
@@ -504,6 +505,7 @@
 		log_target.add_hiddenprint(transferred_by) //log prints so admins can figure out who touched it last.
 		log_combat(transferred_by, log_target, "transferred reagents to", my_atom, "which had [get_external_reagent_log_string(transfer_log)]")
 
+	playsound(my_atom.loc, pick(transfer_sound), 50)
```
Comment (SyncIt21) [inline code/modules/reagents/chemistry/holder/holder.dm:508]: Hell no. This proc is very hot and is called by plumbing factories A LOT. I can't even imagine how painful large plumbing factories are goanna sound.

Move this to sound to play when just transferring reagents between beakers and such

**Review by SyncIt21 — CHANGES_REQUESTED** [review]

**Comment by Loafin34** [comment]
please make it a little bit quieter.

**Comment by DaCoolBoss** [comment]
Some items would benefit from an exception here. Syringes and droppers are the ones that immediately come to mind as things you don't want to hear sloshing from. Sleepy pens too maybe.

File: code/modules/reagents/chemistry/holder/holder.dm [hunk 2]
```diff
@@ -504,6 +504,7 @@
 		log_target.add_hiddenprint(transferred_by) //log prints so admins can figure out who touched it last.
 		log_combat(transferred_by, log_target, "transferred reagents to", my_atom, "which had [get_external_reagent_log_string(transfer_log)]")
 
+
```
Comment (SyncIt21) [inline code/modules/reagents/chemistry/holder/holder.dm:507]:
```suggestion
```

File: code/modules/reagents/reagent_containers/cups/_cup.dm [hunk 3]
```diff
@@ -18,6 +18,7 @@
 	var/gulp_size = 5
 	///Whether the 'bottle' is made of glass or not so that milk cartons dont shatter when someone gets hit by it.
 	var/isGlass = FALSE
+	var/list/transfer_sound = list('sound/effects/liquid_pour1.ogg', 'sound/effects/liquid_pour2.ogg', 'sound/effects/liquid_pour3.ogg')
```
Comment (SyncIt21) [inline code/modules/reagents/reagent_containers/cups/_cup.dm:21]:
```suggestion
	var/static/list/transfer_sound = list('sound/effects/liquid_pour1.ogg', 'sound/effects/liquid_pour2.ogg', 'sound/effects/liquid_pour3.ogg')
```

File: code/modules/reagents/reagent_containers/cups/_cup.dm [hunk 4]
```diff
@@ -18,6 +18,7 @@
 	var/gulp_size = 5
 	///Whether the 'bottle' is made of glass or not so that milk cartons dont shatter when someone gets hit by it.
 	var/isGlass = FALSE
+	var/static/list/transfer_sound = list('sound/effects/liquid_pour1.ogg', 'sound/effects/liquid_pour2.ogg', 'sound/effects/liquid_pour3.ogg')
```
Comment (SyncIt21) [inline code/modules/reagents/reagent_containers/cups/_cup.dm:21]: Move this list to the line 121(where you call play_sound).

It's only used there locally so it doesn't need to be global

Comment (ShizCalev) [inline code/modules/reagents/reagent_containers/cups/_cup.dm:21]:
```suggestion
```

File: code/modules/reagents/reagent_containers/cups/_cup.dm [hunk 5]
```diff
@@ -117,6 +118,7 @@
 			return ITEM_INTERACT_BLOCKING
 
 		var/trans = reagents.trans_to(target, amount_per_transfer_from_this, transferred_by = user)
+		playsound(target.loc, pick(transfer_sound), 50)
```
Comment (ShizCalev) [inline code/modules/reagents/reagent_containers/cups/_cup.dm:121]:
```suggestion
		playsound(target.loc, pick('sound/effects/liquid_pour1.ogg', 'sound/effects/liquid_pour2.ogg', 'sound/effects/liquid_pour3.ogg'), 50)
```

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**